cumulusci/tasks/salesforce/CreateCommunity.py

```diff
@@ -0,0 +1,83 @@
+import json
+import requests
+from datetime import datetime
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.exceptions import SalesforceException
+
+
+class CreateCommunity(BaseSalesforceApiTask):
```

Comment:
Please set `task_docs` and run `cci task doc > docs/tasks.rst` to update the generated task docs.